### PR TITLE
GH claim disk space

### PR DESCRIPTION
**Pull request by pjgg**
Claim some extra disk space to GH in order to avoid errors like :
```
❌ Docker environment should have more than 2GB free disk space
```

### CODE REVIEW

**Comment by rsvoboda** [comment]
@Sgitario do we need the same change also for https://github.com/quarkus-qe/beefy-scenarios/blob/main/.github/workflows/quarkus-snapshot.yaml ? Or it triggers just the JVM run?

**Review by rsvoboda — APPROVED** [review]

**Comment by rsvoboda** [comment]
Jose will prepare the PR for ecosystem ci directly.

File: .github/ci-prerequisites.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Reclaim disk space, otherwise we only have 13 GB free at the start of a job
+
```
Comment (rsvoboda) [inline .github/ci-prerequisites.sh:2]: add `df -h` to see stats about disk usage before cleanup

File: .github/ci-prerequisites.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+time docker rmi node:10 node:12 mcr.microsoft.com/azure-pipelines/node8-typescript:latest
+# That is 18 GB
+time sudo rm -rf /usr/share/dotnet
+# That is 1.2 GB
+time sudo rm -rf /usr/share/swift
```
Comment (rsvoboda) [inline .github/ci-prerequisites.sh:7]: add `df -h` to see stats about disk usage after the cleanup

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: .github/ci-prerequisites.sh (hunk 1 above)
Comment (pjgg) [reply .github/ci-prerequisites.sh:2]: I've also add an echo message

File: .github/ci-prerequisites.sh (hunk 2 above)
Comment (pjgg) [reply .github/ci-prerequisites.sh]: Before: `/dev/root        84G   68G   16G  82% /`
After: `/dev/root        84G   43G   41G  51% /`

File: .github/ci-prerequisites.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Reclaim disk space, otherwise we only have 13 GB free at the start of a job
+
+time docker rmi node:10 node:12 mcr.microsoft.com/azure-pipelines/node8-typescript:latest
+echo "Reclaim disk space."
+df -h
```
Comment (rsvoboda) [inline .github/ci-prerequisites.sh:5]: `df-h` should be before docker rmi execution

**Review by rsvoboda — APPROVED** [review]
Looks good, lets wait for CI to finish

**Pull request merged**